### PR TITLE
Avoid literal 0 in expansions

### PR DESCRIPTION
**Pull request by Amorymeltzer**
There's one more, in ruby-mode's `dow`:

```ruby
downto(${0}) { |${n}|
	$0
}
```

But I don't know what it should be.

### CODE REVIEW

File: snippets/js-mode/pge [hunk 1]
```diff
@@ -6,5 +6,5 @@
 # --
 
 get ${1:propertyName}() {
-  return this.${0}
+  return this.${2:name}
```
Comment (Amorymeltzer) [inline snippets/js-mode/pge:9]: Honestly, I prefer `propertyName` again, but figured it shouldn't be repeated?

Comment (AndreaCrotti) [reply snippets/js-mode/pge:9]: mm if it's supposed to be the same thing you can just use $1 again directly?
If not also just repeating propertyName is fine I don't see the problem.

Comment (Amorymeltzer) [reply snippets/js-mode/pge]: @AndreaCrotti It probably depends on how one has constructed their setup.  It wouldn't necessarily be weird to have different capitalization or a more desirable facing term.  I've gone and repeated `propertyName`, which should hopefully fit the middle-ground.

**Comment by AndreaCrotti** [comment]
Nice thanks

**Pull request merged**